notebooks/ml_fairness_explainability/inclusive_ml/solution/1_inclusive_ml.ipynb

```diff
@@ -32,16 +32,24 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        .set_ai_platform_model(PROJECT, MODEL, VERSION, service_name='ml', adjust_prediction=adjust)
</pre><p>shouldn't this change be copied to the lab version?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>